### ISSUE TITLE
[BUG] - Minimaal layout issue in Fx

### ISSUE THREAD

**Issue by driesva** [milieuinfo/webcomponent-vl-ui-cookie-consent#23]
**Omschrijf het probleem**
Wanneer Fx niet op volledig scherm staat plakt de "Bewaar keuze" knop tegen de onderkant van de modal

**Gewenst gedrag**
White spacing tussen knop en onderkant modal. Werkt correct in Chrome.

**Screenshots**
![image](https://user-images.githubusercontent.com/29454303/76094817-869e3c80-5fc3-11ea-97c9-ec5f025091c6.png)

**Omgeving (gelieve aan te vullen):**
 - Browser Firefox Developer Edition
 - Versie 74.0b9 (64-bit)
 - Versie component 3.0.0

**Extra info**
Geen impact op functionaliteit, gewoon gelogd ter referentie.

**Comment by coemans** [comment]
Dit blijkt geen probleem te zijn met de vl-ui-cookie-consent op zich maar met de vl-ui-modal. Het probleem stelt zich ook met de centrale component.

**Comment by driesva** [comment]
@coemans bestaat er reeds een issue voor de vl-ui-modal?

**Comment by coemans** [comment]
Nee, maar dit issue staat nog open dus kan dienen als reminder voor het modal issue.
